src/_includes/banner.html

```diff
@@ -2,8 +2,8 @@
   For headings use h3 elements.
 {% endcomment -%}
 <div class="site-banner site-banner--default" role="alert">
-  <a href="{{site.main-url}}/global-gamers">
-  Build epic Flutter games to assist in the battle to defend the planet!
-  Join the Global Gamers Challenge.</a>
+  Flutter and Dart’s latest releases are helping to
+  define the future of app development.
+  <a href="">Read the blog</a> to learn more.
```

Comment:
Is this supposed to link to the umbrella post?

```suggestion
  <a href="{{site.flutter-medium}}/starting-2024-strong-with-flutter-and-dart-cae9845264fe">Read the blog</a> to learn more.
```